### ISSUE TITLE
Please update version of dyaml dependency

### ISSUE THREAD

**Issue by denizzzka** [forbjok/yamlserialized#1]
Please update version of dyaml dependency
Latest is 0.7.1

**Comment by dedupely** [comment]
This makes yamlserialized impossible to compile for the latest dmd version.

**Comment by denizzzka** [comment]
currently used dyaml 0.6.5 is broken and fixed only in 0.7.1 (https://github.com/dlang-community/D-YAML/issues/205)
```
$ dub test
Generating test runner configuration 'yamlserialized-test-library' for 'library' (library).
Excluding package.d file from test due to https://issues.dlang.org/show_bug.cgi?id=11847
Performing "unittest" build using /usr/bin/dmd for x86_64.
dunit 1.0.14: target for configuration "library" is up to date.
tinyendian 0.1.2: building configuration "library"...
dyaml 0.6.5: building configuration "library"...
../../.dub/packages/dyaml-0.6.5/dyaml/source/dyaml/node.d(221,9): Error: static assert:  "Unexpected YAML value size"
/usr/bin/dmd failed with exit code 1.
```
